src/npc.h

```diff
@@ -554,7 +554,7 @@ class npc : public player
  ~npc() override;
 
     /** Construct an NPC with standard stats */
-    static npc standard();
+    static npc make_standard();
```

Comment:
An effectively empty `fake_npc` or `standard_npc` class could solve this:
```C++
class fake_npc : public npc {
    using npc::npc;
};

void test() {
    fake_npc my_npc;
    my_npc.do_something();
}
```
It allows using all the constructor of `npc`, but gives it a separate name.